src/store/versionsStore.ts

```diff
@@ -0,0 +1,31 @@
+import { map } from "nanostores";
+import uniqBy from "lodash-es/uniqBy";
+
+import type { Option } from "@adjust/components/build/ComboBox/ComboBox";
+
+interface VersionStore {
+  items: Option[];
+  currentVersion: Option;
+}
+
+export const $versions = map<VersionStore>({
+  items: [],
+  currentVersion: { label: "v5", value: "v5" },
```

Comment:
For now, v4 is still the default version.

```suggestion
  currentVersion: { label: "v4", value: "v4" },
```